README.md

```diff
@@ -448,7 +448,7 @@ lists pre-compiled ``protoc`` artifacts that can be used by this plugin.
 
 ### IntelliJ IDEA
 
-Be sure to [delegate builds to Gradle](https://www.jetbrains.com/idea/whatsnew/#v2016-3-gradle)
+Be sure to enable [Delegating IDE build/run actions to Gradle](https://blog.jetbrains.com/idea/2016/10/intellij-idea-2016-3-eap-refactorings-to-java-8-vcs-integration-and-gradle/)
```

Comment:
The URL doesn't automatically jump to the topic. Maybe just copy `Settings → Build, Execution, Deployment  → Build Tools  → Gradle  → Runner  → Delegate IDE build/run actions to gradle.` into here.